setup.py

```diff
@@ -22,6 +22,9 @@
             "st_referencemaps=shimmingtoolbox.cli.referencemaps:main",
             "st_b0maps=shimmingtoolbox.cli.b0map:main",
             "st_download_data=shimmingtoolbox.cli.download_data:download_data",
+            "st_mask_cube=shimmingtoolbox.cli.mask_cube:mask_cube",
```

Comment:
don't we want to instead deal with a single CLI called `st_mask`, and input the shape as param?

Reply:
Feasible with the click.group command